pyproject.toml

```diff
@@ -13,7 +13,7 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = ">3.8.0,<4"
+python = ">3.8.1,<4"
```

Comment:
Allows flake8 to go to v6.1+ which supports python 3.12. Before that it gets confused by the new f-strings grammar.